app/ai/voice/agents/breeze_buddy/managers/calls.py

```
@@ -187,9 +187,50 @@ async def _retry_call(
+                    success = await send_webhook_with_retry(
+                        session, reporting_webhook_url, summary_data, max_retries=3
+                    )
+                    if success:
+                        logger.info(
+                            f"Successfully sent call summary webhook on no_answer (attempt {lead.attempt_count + 1}, isLastAttempt: {is_last_attempt})."
+                        )
+                    else:
+                        logger.error(
+                            "Failed to send call summary webhook on no_answer after all retries."
+                        )
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)